Doc/library/doctest.rst

```diff
@@ -958,7 +958,7 @@ and :ref:`doctest-simple-testfile`.
 
    Optional argument *exclude_empty* defaults to false.  If true, objects for which
    no doctests are found are excluded from consideration. The default is a backward
-   compatibility hack, so that code still using :meth:`doctest.master.summarize` in
+   compatibility hack, so that code still using :meth:`doctest.master.summarize <DocTestRunner.summarize>` in
```

Comment:
The line's a _bit_ long here:

```suggestion
   compatibility hack, so that code still using
   :meth:`doctest.master.summarize <DocTestRunner.summarize>` in
```